index.html

```diff
@@ -121,31 +121,45 @@
     <section id="why">
         <span id="headline-start">WHY IT WORKS &nbsp</span> <span id="headline-end"> AT A GLANCE</span>
         <div id="subtitle">
-            Boxtribute is made by former coordinators and volunteers who have worked in the field and understand its challenges. It’s designed to work within the flow of an existing team and be ready to go in a month or less.
+            Boxtribute is made by former coordinators and volunteers who have worked in the field and understand its challenges. It’s designed to work within the flow of an existing team and be <strong>ready to go in a month or less.</strong>
         </div>
         <div id="quote">
             “[Boxtribute is] super functional, so you actually don’t need that much training. It speaks [for] itself.” &nbsp <span id="source">— Laura Vigourt, volunteer at Drapen i Havet</span>
         </div>
         <div id="elements">
             <div id="subheader">
-                <div id="title"><strong>SIMPLIFY THE WAREHOUSE</strong></div> With our <strong> smartphone-scannable labels</strong>, aid workers always know what kind of stock they have, how much is available and where it is. With a <strong>searchable list of everything</strong> inside the warehouse, tracking becomes easy!
+                <div id="title">SIMPLIFY THE WAREHOUSE</div> With our <strong> smartphone-scannable labels</strong>, aid workers always know what kind of stock they have, how much is available and where it is. With a <strong>searchable list of everything</strong> inside the warehouse, tracking becomes easy!
             </div>
+        </div>
+
+        <div class="two-by-two-grid">
+
+            <div id="elements" style="padding-bottom: 10%;">
```

Comment:
There are multiple divs with the id "elements". maybe make a class out of it and could there be a better name?
Also please move the styling into a css?

Reply:
Hans, I feel like it doesn't make much sense at this point. The majority of the page is supported by this concept of increasingly more specific element ids. So, I would end up creating a class that is used only once, because "elements" are assuming different roles according to how specific they are. 
Let me know what you think!